server/controllers/availableRooms.js

```diff
@@ -0,0 +1,7 @@
+const getAvailableRooms = require ('../database/queries/getAvailableRooms')
```

Comment:
We talked about this!
NO need for Backend, you should receive the data from rooms component!

Reply:
Yes, I will delete it